site/components/src/main/java/uk/nhs/hee/web/components/Listing.java

```diff
@@ -0,0 +1,97 @@
+package uk.nhs.hee.web.components;
+
+import org.apache.commons.lang.StringUtils;
+
+/**
+ * An enumeration of Listing Types and its document types & category value-list identifier.
+ *
+ * <p>Note that the listing types should be in sync
+ * with the value-list {@code /content/documents/administration/valuelists/listingtypes}.</p>
+ */
+public enum Listing {
```

Comment:
should we use more specific enum name like `ListingPageType` or `ListingPage` because the name `Listing` is quite generic for the new comer

Reply:
@thojava - Will rename as `ListingPage`, thanks

Reply:
Refactored to ListingPageType (including corresponding value-list and field name)